website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -2,21 +2,22 @@
 title: "Python models"
 ---
 
-:::info Beta
+:::info Prerelease
 
-dbt Core v1.3, currently in beta, adds Python models to dbt. Note that only [specific data warehouses](#specific-data-warehouses) support dbt Python models. Support in dbt Cloud will be limited to start, and improve over the course of the beta.
+dbt Core v1.3, currently available as a release candidate, adds Python models to dbt. Note that only [specific data platforms](#specific-data-platforms) support dbt-py models.
 
 We encourage you to:
 - Read [the original discussion](https://github.com/dbt-labs/dbt-core/discussions/5261) that proposed this feature.
 - Contribute to [best practices for developing Python models in dbt](https://github.com/dbt-labs/docs.getdbt.com/discussions/1811).
+- Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).
 - Join the **#beta-feedback-python-models** channel in the [dbt Community Slack](https://www.getdbt.com/community/join-the-community/).
 
-Below, you'll see sections entitled "❓ **Our questions**." We're working to develop opinionated recommendations on how to use this feature, ahead of the final release in October—and you can help! Comment in the GitHub discussions; leave thoughts in Slack; bring up dbt + Python in casual conversation with colleagues and friends.
+Below, you'll see sections entitled "❓ **Our questions**." We are excited to release a first narrow set of functionality in v1.3, which will solve real use cases. We also know this is a first step into a much wider field of possibility. We don't pretend to have all the answers. We're excited to keep developing our opinionated recommendations and next steps for product development—and we want your help. Comment in the GitHub discussions; leave thoughts in Slack; bring up dbt + Python in casual conversation with colleagues and friends.
 :::
 
 ## About Python models in dbt
 
-dbt Python models will help you solve use cases that you can't solve with SQL. You can perform analyses using tools available in the open source Python ecosystem, including state-of-the-art packages for data science and statistics. Before, you would have needed separate infrastructure and orchestration to run Python transformations in production. By defining your Python transformations in dbt, they're just models in your project—with all the same capabilities around testing, documentation, and lineage.
+dbt Python ("dbt-py") models will help you solve use cases that you can't solve with SQL. You can perform analyses using tools available in the open source Python ecosystem, including state-of-the-art packages for data science and statistics. Before, you would have needed separate infrastructure and orchestration to run Python transformations in production. By defining your Python transformations in dbt, they're just models in your project—with all the same capabilities around testing, documentation, and lineage.
```

Comment:
When I see a string of text like `dbt-py`, it makes me think there is an adapter on PyPI for dedicated to Python.

Obviously, the rest of the context here makes it clear that's _not_ what's being discussed, but there's still a bit of dissonance for me.

Not discouraging the use of "dbt-py", just wanted to give you insight into how my 👀 and 🧠 are parsing this sentence.

Reply:
Nice catch! I slipped this in because I was curious to see how it would land for folks

We talk sometimes about dbt-sql / dbt-SQL... maybe dbt-py, too? Maybe this _will_ be the name of a real PyPI packages, in a world where dbt supports "language" plugins? https://github.com/dbt-labs/dbt-core/discussions/5796

Definitely not a necessary inclusion, I don't need to make `dbt-py` happen :)

Reply:
With that context and background, referring to "dbt-py", "dbt-sql", "dbt-prql", "dbt-r", etc makes sense.